co-simulation/fed/mosaic-carma-cloud/src/test/java/org/eclipse/mosaic/fed/carmacloud/ambassador/CarmaCloudRegistrationReceiverTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2023 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+ 
+ package org.eclipse.mosaic.fed.carmacloud.ambassador;
+
+ import static org.junit.Assert.assertEquals;
+ import static org.mockito.Mockito.mock;
+ import static org.mockito.Mockito.when;
+ import org.mockito.internal.util.reflection.FieldSetter;
+
+ import java.io.ByteArrayInputStream;
+ import java.io.ByteArrayOutputStream;
+ import java.io.DataOutputStream;
+ import java.net.InetAddress;
+ import java.net.InetSocketAddress;
+ import java.net.ServerSocket;
+ import java.net.Socket;
+ import java.util.List;
+ 
+ import org.junit.After;
+ import org.junit.Before;
+ import org.junit.Test;
+ 
+ public class CarmaCloudRegistrationReceiverTest {
+ 
+     private static final int TEST_PORT = 1617;
+ 
+//     private Socket sendSocket;
+//     private CarmaCloudRegistrationReceiver receiver;
+ 
+//     @Before
+//     public void setup() throws Exception {
```

Comment:
As mentioned above if these methods are no longer needed, lets remove them. If they are needed lets fix them and uncomment them.

Reply:
This is still an issue.